platform_api/handlers/job_request_builder.py

```diff
@@ -68,7 +68,9 @@ def create_resources_from_payload(payload: dict[str, Any]) -> ContainerResources
         tpu = create_tpu_resource_from_payload(payload["tpu"])
     return ContainerResources(
         cpu=payload["cpu"],
-        memory_mb=payload["memory_mb"],
+        memory=payload["memory"]
+        if "memory" in payload
+        else payload["memory_mb"] * 10**6,
```

Comment:
`10**6` or `2**20`?

Reply:
Thank you, `2**20` will be consistent with current behavior.